app/controllers/verify/finance_controller.rb

```diff
@@ -26,6 +27,16 @@ def create
 
     private
 
+    def submit_idv_form
+      result = idv_form.submit(step_params)
+      analytics.track_event(Analytics::IDV_FINANCE_CONFIRMATION_FORM, result.to_h)
+
+      return if result.success?
+
+      render_failure
```

Comment:
Since this action doesn't communicate with the vendor, what do you think about replacing `render_failure` with `@view_model = view_model` since the first 2 conditions in `render_failure` will always be `false` here.